src/Components/Organization/OrganizationData.js

```diff
@@ -0,0 +1,33 @@
+import React, {useEffect, useState} from 'react';
+import { useHistory } from 'react-router-dom';
+import  { Box, Container, Button} from '@material-ui/core';
+import { getOrganizationInformation } from '../../Services/OrganizationInformation';
+
+const OrganizationData = () => {
+
+    const { push } = useHistory()
+    const [orgData,setOrgData]= useState({})
+
+    useEffect(()=>{
+        const loadData = async () => {
```

Comment:
you should never add a function inside useEffect. you should create it outside the useEffect and call it inside the useEffect

Reply:
Perfect, i already fix it.